lvgl-sys/build.rs

```diff
@@ -62,25 +62,48 @@ fn main() {
         .include(&lv_config_dir)
         .compile("lvgl");
 
-    let cc_args = [
+    let mut cc_args = vec![
         "-DLV_CONF_INCLUDE_SIMPLE=1",
         "-I",
         lv_config_dir.to_str().unwrap(),
         "-I",
         vendor.to_str().unwrap(),
+        "-fvisibility=default",
     ];
 
+    // Set correct target triple for bindgen when cross-compiling
+    let target = env::var("TARGET").expect("Cargo build scripts always have TARGET");
+    let host = env::var("HOST").expect("Cargo build scripts always have HOST");
+    if target != host {
+        cc_args.push("-target");
+        cc_args.push(target.as_str());
+    }
+
+    let mut additional_args = Vec::new();
+    if target.ends_with("emscripten") {
+        if let Ok(em_path) = env::var("EMSDK") {
```

Comment:
If it's not present we could show an error, tell users that they are missing to activate the Emscripten environment.

```
# Activate PATH and other environment variables in the current terminal
source ./emsdk_env.sh
```

Reply:
Fixed in #81